storage/entities/transaction.js

```diff
@@ -144,6 +145,15 @@ const assetAttributesMap = {
 	7: ['asset.outTransfer.dappId', ' asset.outTransfer.transactionId'],
 };
 
+// eslint-disable-next-line no-unused-vars
+const stringToByteOnlyInsert = (value, mode, alias, fieldName) => {
```

Comment:
Why not add this to `utils/inputSerializers.js`?

Reply:
This is a special case for transaction only. That `SELECT` query return a `string` while `INSERT` query expect a Buffer.